src/tests/ftest/nvme/nvme_health.py

```diff
@@ -30,21 +30,18 @@ def test_monitor_for_large_pools(self):
         # pylint: disable=attribute-defined-outside-init
         # pylint: disable=too-many-branches
         no_of_pools = self.params.get("number_of_pools", '/run/pool/*')
-        #Stop the servers to run SPDK too to get the server capacity
-        self.stop_servers()
-        storage = self.get_nvme_max_capacity()
-        self.start_servers()
+        storage = self.get_max_storage_sizes()
 
-        #Create the pool from 80% of available of storage space
-        single_pool_nvme_size = int((storage * 0.80)/no_of_pools)
+        #Create the pool from 90% of available of storage space
+        single_pool_nvme_size = int((storage[1] * 0.90)/no_of_pools)
+        single_pool_scm_size = int((storage[0] * 0.90)/no_of_pools)
```

Comment:
same as above.

Reply:
Updated the test and added fill percentage in yaml.